sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1306,7 +1306,7 @@ def _get_previously_launched_zones() -> Optional[List[clouds.Zone]]:
                 region=to_provision.region,
                 num_nodes=num_nodes,
                 instance_type=to_provision.instance_type,
-                accelerators=to_provision.accelerators,
+                accelerators=to_provision.accelerators,  # type: ignore
```

Comment:
Would be great to remove this if we can just fix the types instead. Why is it needed?

Reply:
I see the red squiggles in vscode as well but I think it's the same on master. `mypy` still is passing for whatever reason on master.
Since it's not related to your PR, can we just remove the `# type: ignore` and address it later?